### DISCUSSION TITLE
How to return path parameter in response.

### DISCUSSION THREAD

**Discussion by PriteshCE01** [microcks/microcks#612] [Q&A] [answered]
suppose I have this path in my json  (which i will upload)
/banking/account/{account_number}/balance'

Now i have 2 question
1) How to make it compatible, whether i am sending anything api should call.
so suppose i am sending /account/1/balance or /account/2/balance.
API call should happen?

2) I want to that account_number in response. How to do that? 
I can not find any documentations.

// here is my json file.
[https://drive.google.com/file/d/15fhVihmerFyPcuXf9OwI5heIKkfEU1Pt/view?usp=sharing](url)

**Comment by lbroudoux** [comment, marked as answer]
Hi 👋 

Please have a look at our OpenAPI mocking guide, there's an illustration of your use-case here: https://microcks.io/documentation/guides/openapi-101/#4-using-path-parameters-in-openapi

Note that you can also reuse path elements in response like described here: https://microcks.io/documentation/using/advanced/templates/#variable-reference-expressions
